spec/path-scanner-spec.coffee

```diff
@@ -136,6 +136,17 @@ describe "PathScanner", ->
         runs ->
           expect(paths).toContain path.join(rootPath, 'dir', 'file7_ignorable.rb')
 
+      it "local directory inclusions don't override global subdirectory exclusions", ->
```

Comment:
Can you make this spec flow more naturally?  The `it` is supposed to be part of the sentence (eg `it 'does stuff correctly'`).